guide/13-business-structure/04-competition-within-the-brt-market.manu

```diff
@@ -16,7 +16,7 @@ until the next tender. If the quality of service is poor, the government has lit
 the contract, which governments are generally very reluctant to do as it involves significant disruption
 of services.
 
-Competition “for the market,” by contrast, traditionally meant ongoing competition for customers for
+Competition “in the market,” by contrast, traditionally meant ongoing competition for customers for
 the same set of services. The traditional downside of competition “for the market” was that firms would
```

Comment:
I just noticed that @aszasz had mentioned something that got lost during the rebase:

> This is right and I think line 20 should replace expression `"for the market"` by `"in the market"` as well... _for_ line 19 and 20 are discussing the same type of competition.